R/LogLogisticOs.R

```diff
@@ -0,0 +1,37 @@
+
+#'  Log-logistic overall survival
+#'
+#' Description - Log-logistic overall survival model object creator
+#'
+#' @export
+LogLogisticOs <- setClass(
+    Class = "LogLogisticOs",
+    contains = "OsModel"
+)
+
+
+#' LogLogisticOs object creator
+#'
+#' Creates a log-logistic overall survival templated object
+#' @rdname LogLogisticOs-class
+#' @importFrom assertthat assert_that
+#' @export
+setMethod(
+    f = "initialize",
+    signature = "LogLogisticOs",
+    definition = function(.Object,
+                          ..., stan = LogLogisticModule(), templated) {
+        callNextMethod(
+            .Object,
+            ...,
+            stan = stan@stan,
+            templated = stan@templated
```

Comment:
Sorry this is confusing me.  The `stan` arugment to initialise should be a `StanModule` object not an OS object. Also in this setup it looks like the `templated` argument is doing nothing ?

Reply:
Regarding the Templated argument. Indeed, it has no role for now. I will delete it for now but here is the motivation behind. Before the OsModel is parametrized, it contains some place holders (eg <link_parameters>) that only exist to "save" space for the link parameters that will be selected later on. When the two sub-model are to be merged, we need to be sure that each and every place holder has been replaced with actual stan code. The templated slot would be `False` as long as there are still place holders in the stan code. It was supposed to become `True` only when the model is ready to be merged. So the idea was that I will work as an inhibitor for non - ready Os models.